In [1]:
import pandas as pd

df = pd.read_csv('filtered/tweets.csv')
pd.set_option('display.max_columns', None)

In [2]:
df.head()

created_at          user_id_str  \
0  Mon Feb 10 07:00:13 +0000 2020            370524875   
1  Mon Feb 10 07:00:31 +0000 2020   992649775584546816   
2  Mon Feb 10 07:01:02 +0000 2020            485922064   
3  Mon Feb 10 07:01:14 +0000 2020  1084156001631252480   
4  Mon Feb 10 07:01:29 +0000 2020            365216346   

                                                text coordinates  \
0  That's the point..of doing what you love to do...         NaN   
1  #Coronavirus Wuhan citizen journalist Chen Qiu...         NaN   
2  Molt bona Informació, clau per entendre on est...         NaN   
3  Yes,\nUgandan Govt shud send them food supplie...         NaN   
4  Update - Sources on the cruise told me the Cap...         NaN   

                                   place_coordinates   place_country  \
0  [[[-0.2160701, 51.5098789], [-0.1301597, 51.50...  United Kingdom   
1  [[[-0.1114755, 51.4194247], [-0.0297308, 51.41...  United Kingdom   
2  [[[2.8954926, 42.2262194], [2.9358833, 42.2262...           Spain   
3  [[[32.1922973, -0.1477888], [32.6836991, -0.14...          Uganda   
4  [[[121.457132, 24.960508], [121.66582, 24.9605...          Taiwan   

  place_country_code                  place_full_name place_type language  
0                 GB               Paddington, London       city       en  
1                 GB               Camberwell, London       city       en  
2                 ES  Avinyonet de Puigventós, España       city       ca  
3                 UG                   Wakiso, Uganda      admin       en  
4                 TW              Taipei City, Taiwan       city       en

In [4]:
df.shape

(452347, 10)

(452347, 10)

In [6]:
df['user_id_str'].value_counts()[0:19]

1216916479             638
365216346              411
853007708814954496     370
32398408               366
1316862031             365
38659844               360
208056970              350
191092262              347
235179462              308
1093066243685392385    208
359612102              194
141383006              193
34533138               183
631142747              181
46717886               179
309374492              178
500825663              178
3193289931             175
17670329               174
Name: user_id_str, dtype: int64

ID counts:

1216916479             638
365216346              411
853007708814954496     370
32398408               366
1316862031             365
38659844               360
208056970              350
191092262              347
235179462              308

In [7]:
df['place_country_code'].value_counts()[0:19]

US    153153
GB     30498
BR     21734
IN     16995
PH     16523
ID     15515
ES     14327
CA     12632
IT     11552
AR      9520
NG      8991
AU      8969
MX      8858
FR      7927
CN      7455
MY      7157
ZA      6386
CO      5862
TR      5749
Name: place_country_code, dtype: int64

tweets per country:

US    153153
GB     30498
BR     21734
IN     16995
PH     16523
ID     15515
ES     14327
CA     12632
IT     11552

In [9]:
df[df['place_country_code']=='US']['place_full_name'].value_counts()[0:19]

Los Angeles, CA      5232
Manhattan, NY        4259
Florida, USA         2479
Washington, DC       2452
Houston, TX          2329
Chicago, IL          2278
San Francisco, CA    2201
Brooklyn, NY         2165
Georgia, USA         1945
Texas, USA           1756
Seattle, WA          1607
Queens, NY           1500
San Antonio, TX      1352
Pennsylvania, USA    1341
Austin, TX           1286
California, USA      1250
Philadelphia, PA     1194
San Diego, CA        1169
Atlanta, GA          1091
Name: place_full_name, dtype: int64

tweets per US city:

Los Angeles, CA      5232
Manhattan, NY        4259
Florida, USA         2479
Washington, DC       2452
Houston, TX          2329
Chicago, IL          2278
San Francisco, CA    2201
Brooklyn, NY         2165
Georgia, USA         1945
Texas, USA           1756
Seattle, WA          1607
Queens, NY           1500
San Antonio, TX      1352
Pennsylvania, USA    1341
Austin, TX           1286
California, USA      1250
Philadelphia, PA     1194
San Diego, CA        1169
Atlanta, GA          1091

In [11]:
df['language'].value_counts()[0:19]

en     272112
es      53569
und     31735
pt      21357
in      17352
it       9756
tl       8837
fr       8603
tr       5612
de       3112
ja       2953
nl       2563
ca       1792
hi       1684
zh       1477
ar       1306
th       1219
lt        916
ht        649
Name: language, dtype: int64

tweets per written language:
en     272112
es      53569
und     31735
pt      21357
in      17352